runtime/dancebox/src/lib.rs

```diff
@@ -2607,6 +2611,62 @@ impl_runtime_apis! {
             XcmCoreBuyer::is_core_buying_allowed(para_id)
         }
     }
+
+    impl xcm_fee_payment_runtime_api::XcmPaymentApi<Block> for Runtime {
+        fn query_acceptable_payment_assets(xcm_version: staging_xcm::Version) -> Result<Vec<VersionedAssetId>, XcmPaymentApiError> {
+            if !matches!(xcm_version, 3 | 4) {
+                return Err(XcmPaymentApiError::UnhandledXcmVersion);
+            }
+
+            Ok([VersionedAssetId::V4(xcm_config::SelfReserve::get().into())]
+                .into_iter()
+                .chain(
+                    pallet_asset_rate::ConversionRateToNative::<Runtime>::iter_keys().filter_map(|asset_id_u16| {
```

Comment:
why is this not implemented in the container-chain templates?

Reply:
Fixed